go/vt/topotools/routing_rules.go

```diff
@@ -27,7 +27,7 @@ import (
 	vschemapb "vitess.io/vitess/go/vt/proto/vschema"
 )
 
-//region routing rules
+// region routing rules
```

Comment:
Why do we have these region ... endregion in this file?

Reply:
IDEs support code folding through these directives, which I've been utilizing to group logically related sections of code together.